frontend/src/types.ts

```diff
@@ -2730,6 +2730,7 @@ export interface SurveyAppearance {
     thankYouMessageCloseButtonText?: string
     autoDisappear?: boolean
     position?: string
+    zIndex?: string
```

Comment:
does this field already exist in the `SurveyAppearance` in posthog-js?

Reply:
Yeah, here https://github.com/PostHog/posthog-js/blob/26150b197dc1a2f5241b8d29ee7a359331a1a0a4/src/extensions/surveys/surveys-utils.tsx#L29 

and here : 
https://github.com/PostHog/posthog-js/blob/26150b197dc1a2f5241b8d29ee7a359331a1a0a4/src/posthog-surveys-types.ts#L39

Reply:
huh, so weird we didn't have this capability already.